### ISSUE TITLE
Allow account deletion

### ISSUE THREAD

**Issue by groggemans** [ansible/galaxy-issues#58]
Add the option to delete your account.

Reduces inactive users, allows to connect github account with a new user, ...

**Comment by gregdek** [comment]
Yep. We'll also likely force the deletion of uploaded roles as well. Will be necessary if we do force the move to Github namespaces as well.

**Comment by kmf** [comment]
+1

**Comment by jeremyn** [comment]
It's unfortunate that we still can't delete accounts. I've submitted https://galaxy.ansible.com to Account Killer at https://www.accountkiller.com/en/Submit and referenced this issue. Account Killer is a site where you can look up how to delete your account at various places or find out ahead of time that accounts can't be deleted. I encourage anyone else interested in this issue to submit Ansible Galaxy to Account Killer or any other related site. (I'm not affiliated with Account Killer, just to be clear.)

I'm not sure what else to do about this.
